### DISCUSSION TITLE
component repository : do webhooks only work with https:// repository addresses?

### DISCUSSION THREAD

**Discussion by emmapeel2** [WeblateOrg/weblate#10477] [General] [answered]
I have the feeling that the webhooks in weblate only work when the repository url setting is the https://gitlab.... kind, but they do not work if you add the repo as git:// or ssh:// . 

Is this true? Maybe we could add a note when configuring the repositories. I like to add the https:// repositories because then people can access the repo with one click on the component/info page, and because I think the webhooks dont work otherwise.

Also maybe a note on the webhooks section of the documentation?

**Comment by nijel** [comment, marked as answer]
It should work for ssh URLs. git ones are probably supported as well, but not really maintained as the protocol is insecure.
